### PR TITLE
Add support for Loaders Modernized

### PR DESCRIPTION
**Pull request by kryojenik**

### CODE REVIEW

File: locale/ca/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/ca/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/de/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/de/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/es-ES/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/es-ES/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/fr/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/fr/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/ja/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/ja/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/pt-BR/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/pt-BR/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/ru/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/ru/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

File: locale/uk/prismatic-belts-locale.cfg
Comment (kirazy) [inline locale/uk/prismatic-belts-locale.cfg:1]: Remove this change, all locale files other than English are auto-generated by the Factorio Mod Translation project, and will be lost on the next update.

**Review by kirazy — CHANGES_REQUESTED** [review]
Other than the non-English locale changes, looks good.

**Comment by kryojenik** [comment]
Thanks, updated to remove translation updates in non-English locales.

**Comment by kirazy** [comment]
Oh you should be aware that heating does not currently work correctly for loaders.

https://forums.factorio.com/viewtopic.php?f=65&t=117803

**Pull request merged**